tests/testMaskNans.py

```diff
@@ -20,22 +20,18 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-import sys
 import unittest
 
 import numpy
-import lsst.utils.tests as tests
+import lsst.utils.tests
 import lsst.afw.image as afwImage
 import lsst.afw.display.ds9 as ds9
 from lsst.ip.isr import maskNans
 
-try:
-    debug
-except NameError:
-    debug = False
+debug = False
```

Comment:
Again, confused as to why it was done this way. And hence not sure I like the new solution.